examples/data-sources/hpegl_metal_usage/data-source.tf

```diff
@@ -0,0 +1,22 @@
+// (C) Copyright 2020-2022 Hewlett Packard Enterprise Development LP
+
+variable "start" {
+  default = "2020-01-13T07:44:00Z"
+}
+
+variable "end" {
+  default = "2020-04-13T07:44:00Z"
+}
+
+data "hpegl_metal_usage" "used" {
+  start = var.start
+  #end = var.end
```

Comment:
Is end optional?  Add comment or remove?

Reply:
Yes, optional. Removed it.